src/online2bin/online2-tcp-nnet3-decode-faster.cc

```diff
@@ -248,7 +284,13 @@ int main(int argc, char *argv[]) {
 
           Vector<BaseFloat> wave_part = server.GetChunk();
           feature_pipeline.AcceptWaveform(samp_freq, wave_part);
+
+          // dump 1st chunk of 'raw' audio (debug),
```

Comment:
I feel like a lot of these vlog statements are debug things that probably shouldn't remain.